framework/include/eventloop/eventloop.h

```diff
@@ -165,23 +176,34 @@ int eventloop_delete_timer(el_timer_t *timer);
  * @param[in] repeat the value which represents whether timer runs repeatedly or not. \n
  *            You can stop the periodic timer by calling eventloop_delete_timer
  * @param[in] func the callback function to be called
+ *            In case of repeated timer, it should return EVENTLOOP_CALLBACK_STOP(false) or EVENTLOOP_CALLBACK_CONTINUE(true).
  * @param[in] cb_data data to pass to func when func is called
  * @return On success, A pointer of created timer is returned. On failure, NULL is returned
  * @since TizenRT v2.1 PRE
  */
 el_timer_t *eventloop_add_timer_async(unsigned int timeout, bool repeat, timeout_callback func, void *cb_data);
 
+/**
+ * @brief Set mode of timer
+ * @details @b #include <eventloop/eventloop.h> \n
+ * The mode of all timer is EL_MODE_DEFAULT. \n
+ * A timer with EL_MODE_DEFAULT mode is valid when task/thread which registered this timer is alive. \n
+ * In case of EL_MODE_WAIT_DONE, task/thread which registered this timer doesn't exit, waits for registered events and processes them. \n
```

Comment:
How about `In case of EL_MODE_WAIT_DONE, a task/thread which registered this timer doesn't exit, and the task/thread waits for registered timers and processes them.`?